resources/assets/js/management/roles/components/RolesListing.vue

```diff
@@ -83,10 +115,60 @@ export default {
           name: "__slot:actions",
           title: ""
         }
-      ]
+      ],
+      name: '',
+      code: '',
+      description: '',
+      uid: '',
+      status: '',
+      curIndex: '',
     };
   },
   methods: {
+    onAction(action, data, index) {
+      switch(action) {
+        case 'edit-item' :
+          this.showEditModal(data,index)
+      }
+    },
+    showEditModal(data, index) {
+      this.name = this.data.data[index].name;
+      this.code = this.data.data[index].code;
+      this.status = this.data.data[index].status;
+      this.description = this.data.data[index].description;
+      this.uid = this.data.data[index].uid;
+      this.curIndex = index;
+      this.$refs.editItem.show();
+    },
+    hideEditModal() {
+      this.$refs.editItem.hide()
+    },
+    submitEdit(rowIndex) {
+      window.ProcessMaker.apiClient.post('roles/update/' + this.uid, {
+        'uid': this.uid,
+        'name': this.name,
+        'code': this.code,
+        'description': this.description,
+        'status': this.status
+      })
+      .then((response) => {
+        ProcessMaker.alert("Saved", "success")
```

Comment:
Where are you updating the list data set?

Reply:
above .then, I am setting uid, name etc equal to the bound data attributes. is that not updating it? if not, why am I able to see the changes?